merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/engine/SimulationEngine.java

```diff
@@ -50,6 +52,11 @@
  * A representation of the work remaining to do during a simulation, and its accumulated results.
  */
 public final class SimulationEngine implements AutoCloseable {
+
+  private static final Logger logger = LoggerFactory.getLogger(SimulationEngine.class);
+
+  private static final boolean PROFILE_RESOURCES = false;
```

Comment:
It would be nice to be able to configure this with an env var like `MERLIN_WORKER_PROFILE_RESOURCES`